docs/reference/ilm/actions/ilm-allocate.asciidoc

```diff
@@ -0,0 +1,118 @@
+[[ilm-allocate]]
+=== Allocate
+
+Phases allowed: warm, cold.
+
+The Allocate action allows you to specify which nodes are allowed to host the
+shards of the index and set the number of replicas.
+Behind the scenes, it is modifying the index settings
+for shard filtering and/or replica counts. When updating the number of replicas,
+configuring allocation rules is optional. When configuring allocation rules,
+setting number of replicas is optional. Although this action can be treated as
+two separate index settings updates, both can be configured at once.
+
+For more information about how {es} uses replicas for scaling, see
+<<scalability>>. See <<shard-allocation-filtering>> for more information about
+controlling where Elasticsearch allocates shards of a particular index.
+
+--
+NOTE: As allocate action is not allowed in `hot` phase, the initial allocation for the index should be done manually or via index templates, as ILM won't take care of index allocation during `hot` phase.
+--
+
+[[ilm-allocate-options]]
+.Allocate Options
+[options="header"]
+|======
+| Name                 | Required  | Default     | Description
+| `number_of_replicas` | no        | -           | The number of replicas to
+                                                   assign to the index
+| `include`            | no        | -           | assigns an index to nodes
+                                                   having at least _one_ of the attributes
+| `exclude`            | no        | -           | assigns an index to nodes having
+                                                   _none_ of the attributes
+| `require`            | no        | -           | assigns an index to nodes having
+                                                   _all_ of the attributes
+|======
```

Comment:
These should be converted to a definition list:

```
`number_of_replicas`::
(Optional, integer)
Number of replicas to assign to the index.
```

I also think `properties` may be a more appropriate term than `options`, but I don't feel strongly.

Reply:
Yeah, good opportunity to get rid of the table altogether.